import-automation/progress-dashboard-rest/app/main.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Entry point of the API.
+When deployed on App Engine, gunicorn will serve the FLASK_APP variable,
+which is a Flask object.
+"""
+
+import os
+
+import flask
+import flask_restful
+
+from app.resource import import_attempt, import_log
+from app import utils
+
+
+def create_app(logging=True):
+    """Creates the Flask app.
+
+    Args:
+        logging: Whether to set up Google Cloud Logging.
+    """
+    if not logging:
```

Comment:
Is the choice between Google Cloud Logging and Python logging?  Xor is this logic accidentally reversed?

Reply:
Ops. It is reversed.

Reply:
fixed